### PR TITLE
[debug] raise remind when git-lfs not available

### PR DESCRIPTION
**Pull request by boji123**

### CODE REVIEW

**Comment by boji123** [comment]
@csukuangfj

File: egs/aishell/ASR/prepare.sh [hunk 1]
```diff
@@ -48,6 +48,8 @@ if [ $stage -le -1 ] && [ $stop_stage -ge -1 ]; then
   log "stage -1: Download LM"
   # We assume that you have installed the git-lfs, if not, you could install it
   # using: `sudo apt-get install git-lfs && git-lfs install`
+  git lfs2 1>/dev/null 2>&1 || (echo "please install git-lfs, consider using: sudo apt-get install git-lfs && git-lfs install" && exit 1)
```
Comment (csukuangfj) [inline egs/aishell/ASR/prepare.sh:51]:
```suggestion
  git lfs 1>/dev/null 2>&1 || (echo "please install git-lfs, consider using: sudo apt-get install git-lfs && git-lfs install" && exit 1)
```

**Comment by boji123** [comment]
@csukuangfj solved

**Comment by csukuangfj** [comment]
Thanks!

**Pull request merged**